package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "app.js",
   "scripts": {
+    "start": "nodemon -r esm ./app.js",
```

Comment:
please use `nodemon` with `@babel/register` to run `app.js`

Reply:
Updated the start script to use @babel/register